src/content/whats-new/2024/12/whats-new-12-06-AWS-logs.md

```diff
@@ -0,0 +1,15 @@
+title: 'Forwarding logs from AWS services to New Relic’
+summary: 'New Relic snow supports forwarding logs from AWS services.’
+releaseDate: '2024-12-09'
+learnMoreLink: 'https://deploy-preview-19463--docs-website-netlify.netlify.app/install/aws-logs/?service=aws_services'
+getStartedLink: 'https://deploy-preview-19463--docs-website-netlify.netlify.app/docs/logs/get-started/get-started-log-management/’
+---
+# Forwarding logs from AWS services to New Relic
+New Relic now supports forwarding logs from AWS services. Our AI-powered log management solution provides instant insights from contextualization with all telemetry data across your entire stack to optimize operations and business outcomes. Forwarding logs from AWS Services amplifies your visibility into application and infrastructure performance data (events, errors, traces, and more) to reduce mean time to resolution (MTTR) and quickly troubleshoot production incidents. Our AI-powered parsing and recommendations give you unparalleled visibility into your system's performance so you catch issues before they become problems, and make data-driven decisions with ease. Enhance your team's operational efficiency while maintaining security and compliance.
+To get started:
```

Comment:
**Get started today** to achieve a unified, holistic view of your AWS environment, enabling data-driven decision-making while ensuring security and compliance. Discover how New Relic's logs management can boost productivity and help you securely scale your cloud-native applications.